README.md

```diff
@@ -1049,6 +1050,61 @@ kubectl logs <v1 pod> -n testns1 -c istio-proxy -f --tail 0
 [2020-11-23T10:52:12.711Z] "GET /delay/15 HTTP/1.1" 0 DC "-" "-" 0 0 1999 - "-" "curl/7.35.0" "fc786864-ce72-4eae-9e20-9d9df1ac26ac" "httpbin.testns1.svc.cluster.local" "127.0.0.1:80" inbound|8000|http|httpbin.testns1.svc.cluster.local 127.0.0.1:45190 192.168.94.99:80 192.168.64.239:55220 outbound_.8000_.v1_.httpbin.testns1.svc.cluster.local default
 ```
 
+#### retries (baseInterval and maxInterval)
+The retry interval can be controlled by setting envoyfilter.
+In the following configuration, base_interval is set to 4S and max_interval is set to 5s.
```

Comment:
In the following configuration, base_interval is set to 4s and max_interval is set to 5s.